packages/init/src/steps/install-required-dependencies.ts

```diff
@@ -2,6 +2,27 @@ import path from "path";
 import { execCommand } from "../utils";
 import { Utils, FileSystem } from "@slicemachine/core";
 
+function depsForFramework(framework: Utils.Framework.FrameworkEnum): string {
+  const packages = [];
```

Comment:
OPT: I think a switch with the frameworks would be better, to have a clearer Code.

```
switch(framework) {
  case(Framework.next): ([SM_PACKAGE_UI, PRISMIC_PRISMIC_PACKAGE_REACT, PRISMIC_PACKAGE_CLIENT]).join(" ")
}
```

Or something similar 😄

Reply:
switches fall through, i did try them. I guess it can have unique cases that return.

Reply:
I think the switch makes clear what are the dependencies by framework but create some redundant code. What is the most to you @MarcMcIntosh @Heyrows ?